api/__tests__/integration/seriesService.int.test.ts

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+import {createConnection, Connection, QueryBuilder} from "typeorm";
+import supertest, {SuperTest, Test} from 'supertest';
+import {createServer} from "http";
+
+import app from '../../app';
+import DatabaseConfig from '../../database/config';
+import SeriesModel from "../../series/model";
+
+let request: SuperTest<Test>;
+let dbConnection: Connection;
+let queryBuilder: QueryBuilder<any>;
+
+beforeAll(async () => {
+    request = supertest(createServer(app.callback()));
+    dbConnection = await createConnection(DatabaseConfig);
+    queryBuilder = await dbConnection
+        .createQueryBuilder();
+});
+
+afterAll(async () => {
+    await dbConnection.close();
+});
+
+// Thanks to https://stackoverflow.com/a/61648442/795407
+describe('Series Service', () => {
+
+    describe('Requires existing series', () => {
+
+        let seriesId;
+
+        beforeEach(async () => {
+            const seriesInsertResult = await queryBuilder.insert()
+                .into(SeriesModel)
+                .values([
+                    {name: 'test series'}
+                ])
+                .execute();
+
+            seriesId = seriesInsertResult.identifiers[0].id;
+        });
+
+        afterEach(async () => {
+            await queryBuilder.delete()
+                .from('series')
+                .whereInIds([seriesId])
+                .execute()
+        });
+
+        it('should list series', async () => {
+            const response = await request.get('/series')
+                .expect(200);
+
+            expect(response.body).toHaveProperty('series');
+            // We have two series from seed data
+            // We may want to remove these later...?
+            expect(response.body.series.length).toBe(3);
```

Comment:
Randomly, this test case fails at this line complaining the length of the series is 4 instead of 3.

It looks like this is due to jest runs test cases in parallel as default. It is not the best practice but, maybe we should run the integration test in serial order by `jest int -i` and update test scripts in package.json?

Some references that I found.
https://stackoverflow.com/questions/49937456/randomly-failing-tests-jest-and-supertest-node-js
https://stackoverflow.com/questions/39831961/are-tests-inside-one-file-run-in-parallel-in-jest

Reply:
Ah, damn. I forgot about this. Thank you. Maybe it would be better to just confirm that at least the two test events exist. This point is to make sure that events we expect to be returned are returned, so extra events are no problem. I'll update it. Good find!

Reply:
Sounds good, and LGTM. Thanks for setting it up!

Reply:
I've fixed the issue. Please confirm?

I want to make sure things are in a good state before merging. I'd like the master branch to be as stable as possible. Sorry for the delay. Busy times with girlfriend and her family.

Reply:
Ah, and to directly answer your question, we should merge now that it's stable.

Reply:
LGTM and I'm merging :)